tests/UnitTests.cpp

```diff
@@ -0,0 +1,20 @@
+#include "gtest/gtest.h"
+#include "../JsonParser.h"
+
+TEST(ParserTest, CheckReturnedMapSize)
+{
+    std::ifstream f1("../units/vader.json");
+    EXPECT_EQ(3, JsonParser::Parse(f1).size());
+
+    std::ifstream f2("../units/luke.json");
+    EXPECT_EQ(3, JsonParser::Parse(f2).size());
+
+    std::ifstream f3("../units/palpatine.json");
+    EXPECT_EQ(3, JsonParser::Parse(f3).size());
+}
```

Comment:
"Ezek utan keszuljon 3 unit teszt erre az uj parserre."
Ez 1 teszteset, nem 3, még ha mindjárt 3 fájlt nyitsz is meg benne.
1) attól, hogy három kulcs-érték pár lesz az eredmény, ezek még lehetnek hibás tartalmúak, vagy lehetnek olyan kulcsok, amiket a unit nem tud használni semmire.
2) Minden json fájl teljesen azonos szerkezetű. Nincs egyikben sem semmiféle hiba, pl. hiányzó kulcs, szükségtelen kulcs, kimaradt : karakter, stb.
3) A 3 elkészült fv.-ből mindig ugyanazt az egyet teszteled.